docs/framework-app_dynamics_agent.md

```diff
@@ -17,14 +19,14 @@ When binding AppDynamics using a user-provided service, it must have name or tag
 
 | Name | Description
 | ---- | -----------
```

Comment:
Hi Ben,

We currently do-not support auto-provisioning mechanism where if the account-access-key, account-name is not specified then we automatically provision an account for such users. Currently such mechanism is not supported by AppDynamics

Hence, parameters such as account-access-key, account-name, port, ssl-enabled, host-name are required.

However, since application-name, tier-name and node-name are automatically populated using the logic mentioned in java-buildpack, these parameters can be set as optional in the doc file.